languages/rust/exercises/concept/string-vs-str/.meta/design.md

```diff
@@ -0,0 +1,50 @@
+# Goal
+
+Teach the student the differences between String and &str and how to work successfully with both.
+Learning objectives
+
+    Know String is owned and mutable
+    Know &str is shared and immutable
+    Know String has &str methods because of shared traits
+    Know of performance characteristics
+
+## Out of scope
+
+    Machine level representation of either
+    in depth performance differences
+
+## Concepts
+
+    mutability
+    ownership
+    slices
+
+## Prerequisites
+
+    basics
+    slices
+    str-use
+    string-use
+
+## Resources to refer to
+
+## Hints
+
+    [string slices discussion in TRPL book][https://doc.rust-lang.org/book/ch04-03-slices.html#string-slices]
```

Comment:
```suggestion
    (string slices discussion in TRPL book)[https://doc.rust-lang.org/book/ch04-03-slices.html#string-slices]
```

Reply:
I think you replaced the wrong set of brackets with parentheses.

It happens to everyone.